features/step_definitions/landing_page_steps.rb

```diff
@@ -0,0 +1,20 @@
+When(/^I visit the LandingPage$/) do
+  landing_page.load
+end
+
+Then(/^I want to be able to input my age$/) do
+  expect(page).to have_css('#your_details_form_age')
+end
+
+And(/^I want to be able to input my gender$/) do
+  expect(page).to have_css('#your_details_form_gender')
```

Comment:
In order to ensure that we are referring to the same page that we navigated to in the first step `When I visit the Landing Page`. Here we should seek our expectations from the `landing_page` object.